packages/venia-ui/lib/targets/venia-ui-intercept.js

```diff
@@ -43,5 +45,19 @@ module.exports = veniaTargets => {
             '@magento/venia-ui/lib/components/SavedPaymentsPage/creditCard'
     });
 
+    const editablePayments = new EditablePaymentTypes(venia);
```

Comment:
Instead of four different collections that are all keyed by `paymentCode`, can we simplify these into a config object? Rough idea of that structure.

```js
const paymentConfig = {
  code: 'braintree',
  checkout: {
    list: '<import statement>',
    edit: '<import statement>',
    summary: '<import statement>'
  },
  customer: {
    vault: '<import statement>'
  }
}
```